Test/main.cpp

```diff
@@ -0,0 +1,110 @@
+#include <iostream>
+#include <fstream>
+
+std::string name = "",email = "",passcode = "",time_zone = "",file_path = "";
```

Comment:
Dangerous global variables that are not const could lead to data side-effects an difficult debugging. These should be  within you main function or const.